text/0685-new-browser-support-policy.md

```diff
@@ -0,0 +1,293 @@
+polyfills can still be provided by Ember applications, and may continue to work -
+they are just not explicitly supported.
+
+## What support means
+
+This policy governs two major aspects of Ember and Ember Data:
+
+1. When the framework adopts new browser features
+2. How the framework responds to bug reports
+
+Ember and Ember Data _may_ introduce new browser feature usage in any _minor_
+version release of Ember, provided the feature is supported in all browsers that
+are supported in this policy at the time the the version in released.
+
+Ember and Ember Data _may_ introduce new browser feature usage in any _patch_
+version release of Ember, provided the feature is supported in all browsers that
+were supported in this policy at the time the _minor_ that the patch is applied
+to was released. In other words, no breakage can or should occur due to new
+browser feature usage in _any_ patch release, and if it does, it is a bug.
+
+For bug reports, support is determined by combining our existing Ember version
+support policy with the browser versions that were supported at the time an
+Ember version was released. This means Ember will work as time and resourcing is
+available to fix any browser specific issues that occur in:
+
+1. The current stable and LTS releases of Ember and Ember Data
+2. For browsers that were supported by those versions when they were relased
+
+For the previous LTS release, only security bugfixes will be supported,
+following the existing LTS policy.
+
+## Future changes to the policy
+
+This policy is not meant to be immutable. Over time, new browsers could be added
+to the support matrix, and explicit exceptions could be added for "critical
+internet infrastructure".
+
+For example, it was well known that Google's search crawler used a very old
+version of Google Chrome for a very long time. It no longer does and is
+regularly updated, but during that time frame it was important to support that
+version of Chrome. Likewise, it was very important to support IE11 for a very
+long time since it had a large usage percentage, especially in corporate
+environments. While none of these cases are known to exist at the time of this
+writing, if one should arise it may be added explicitly via RFC.
+
+Future RFCs may amend this policy in a strictly additive way without requiring
+a major version bump in Ember. Newly supported browsers will begin being
+supported in the next minor version of Ember after the updated support policy is
+implemented.
+
+Future RFCs that amend this policy to _remove_ support for a browser will
+_require_ a major version of Ember to implement.
+
+## Implementation
+
+This policy drops support for a major browser, and therefore can only be
+implemented with a major version bump.
+
+## How we teach this
+
+- A blog post explaining the policy with a link to this RFC will be posted.
+- Browser support policy will be linked in repo
```

Comment:
What is the actual policy? This RFC likely contains all the information, but it doesn't seem like it is quite in the format that we would want?